install/kubernetes/helm/istio/charts/ingress/templates/clusterrolebinding.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: istio-pilot-{{ .Release.Namespace }}
+  name: istio-ingress-{{ .Release.Namespace }}
```

Comment:
I should add a comment here...  Unfortunately this kind of changes tend to break upgrade from 0.8 - we need to be very careful with that. How about tackling this in master, and make sure we have a way to handle the upgrade (may require separate service account or other tricks).

Reply:
You mean `helm upgrade` or upgrade with `kubectl apply ...`? 
Helm upgrading for istio currently broken because of an helm issue: https://github.com/helm/helm/issues/1193
But I think upgrading istio with `kubectl apply ...` will be OK.

Reply:
@costinm I did not see any issue when upgrade with this change, can you please show more detail for the upgrade issue?